.nvmrc

```diff
@@ -1 +1 @@
-4.4
+8.9
```

Comment:
Line 22 of the README.md has wrong instructions about node and NPM. 

Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) to keep multiple Node versions on your system.


PLUS but not blocking:
I think we also could add a little doc on upgrading from node 4.4 (running nvm install again, and npm install, or even other forms if you dont use nvm)

Reply:
Good catch, got it updated 👍